src/extras/PMREMGenerator.js

```diff
@@ -409,12 +409,6 @@ class PMREMGenerator {
 
 		uniforms[ 'envMap' ].value = texture;
 
-		if ( ! isCubeTexture ) {
-
-			uniforms[ 'texelSize' ].value.set( 1.0 / texture.image.width, 1.0 / texture.image.height );
```

Comment:
I realized my previous simplification of PMREM had made this vestigial, so I removed it as well.